python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -369,6 +368,11 @@ def accept(self):
         if self.chkSpatialIndex.isEnabled() and self.chkSpatialIndex.isChecked():
             self.db.connector.createSpatialIndex((schema, table), geom)
 
+        # add comment on table
+        if self.chkCom.isEnabled() and self.chkCom.isChecked():
```

Comment:
What's the `isEnabled()` part for?

Reply:
I think the two things are unrelated.
Even if it's disabled, the checked state should represent what the user can expect.

Reply:
Hmm i don't really know. I was just guessing. But probably the thing is if it is disable, the state value of the checkBox is not needed. This because the user only should be the one who decide.
Actually i just use the same syntax as the geom or primary key widget.

Reply:
If it's disabled the checked state should be set correctly by the app. Otherwise it's a miscommunication of the real effect towards the user.

Reply:
Do you know if it actually ever *is* disabled?

Reply:
I don't think it would ever be disabled.
As i said, i just used the same syntax as the one used for other lines. So for me it is just a safe check that prevent misscomunication. But in fact, it might be possible to drop it without any consequences.

Reply:
Ok, let's leave it as is to be equal to the other widgets and assume that someone introduced it for some (obscure) good reason.